cmd.go

```diff
@@ -77,17 +75,21 @@ func (r *Runner) SetupRoot(cmd *cobra.Command) *Runner {
 }
 
 // Setup sets up the Command
-func (r *Runner) Setup(cmd *cobra.Command, runner interface{}, cfg interface{}) *Runner {
+func (r *Runner) Setup(cmd *cobra.Command, runner fx.Option) *Runner {
```

Comment:
To keep this from being a breaking change, let's add back `, _ interface{}`

Reply:
Added the `_ interface{}` back in `Run(..)`. As `Setup` is called by `Run`, we might not need it here. Or is it being used in other places? 🤔

Reply:
Yes it is used in other places.